core/src/plugins/filed/python/pyfiles/BareosFdPluginLocalFileset.py

```diff
@@ -132,3 +132,38 @@ def start_backup_job(self):
             return bareosfd.bRC_Error
         else:
             return bareosfd.bRC_OK
+
+    def get_acl(self, acl):
+        bareosfd.DebugMessage(
+            100, "my get_acl() entry point in Python called with %s\n" % (acl)
+        )
+        acl.content = bytearray(b"Hello ACL")
+        return bareosfd.bRC_OK
+
+    def set_acl(self, acl):
+        # bareosfd.DebugMessage(
+        #     100, "my set_acl() entry point in Python called with %s\n" % (acl.content)
+        # )
+        # bareosfd.JobMessage(
+        #     bareosfd.M_INFO,
+        #     "my set_acl() entry point in Python called with %s\n" % (acl.content)
+        # )
+        return bareosfd.bRC_OK
+
+    def get_xattr(self, xattr):
```

Comment:
what about multiple xattr values? Does that work, too? How?

Reply:
Yes it does by returining bRC_more. The current version has a test for that now.